asts/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import * as ts from 'typescript/lib/tsserverlibrary';
+import * as mm from 'minimatch';
+import * as path from 'path';
+
+let DEFAULT_CONFIG = { include: "assembly/**/*.ts", rootPath: '' };
+let PROJECT: ts.server.Project | null = null;
```

Comment:
I could've used a class here, just aiming to keep things simple in the extension. If there are more features that need to be added here, I'll transform it into a class.